src/runtime-rs/crates/hypervisor/src/ch/inner_hypervisor.rs

```diff
@@ -489,6 +489,18 @@ impl CloudHypervisorInner {
         caps.set(CapabilityBits::FsSharingSupport);
         Ok(caps)
     }
+
+    pub(crate) fn set_capabilities(&mut self, _flag: CapabilityBits) {
+        todo!()
```

Comment:
I guess this is a WIP PR?

Reply:
It's enabled for Dragonball, left `ch` and `qemu` as todo